scripts/set_env_vars.sh

```diff
@@ -21,8 +21,9 @@ export PACT_BROKER_BASIC_AUTH_USERNAME=
 export PACT_BROKER_BASIC_AUTH_PASSWORD=
 export PACT_CONSUMER=nhsd-apim-eps-test-client
 export PACT_PROVIDER=nhsd-apim-eps-sandbox
-export APIGEE_ENVIRONMENT=internal-dev-sandbox
 export APIGEE_ACCESS_TOKEN=
-export SERVICE_BASE_PATH=electronic-prescriptions-pr-171
-export BUILD_VERSION=AEA-555_refactor-examples+1
-export COMMIT_SHA=8f70bfd1976924c3533bc6f5977d075786e873f8
\ No newline at end of file
+export APIGEE_ENVIRONMENT=internal-dev-sandbox
+export SERVICE_BASE_PATH=electronic-prescriptions-pr-189
```

Comment:
If this is a placeholder to show how to run pacts, is it worth adding a section in the `running tests` section of the readme explaining that this needs setting?

Reply:
that would probably be helpful, i'll add this comment to the ACs for the ticket i'm working on now